### PR TITLE
Update development and maintainer document

### PR DESCRIPTION
**Pull request by junqiu-lei**
Signed-off-by: Junqiu Lei <junqiu@amazon.com>

### Description
* Update development document for how to run, test and build plugin.

* Update current maintainers.

### Issues Resolved
https://github.com/opensearch-project/dashboards-maps/issues/229

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: MAINTAINERS.md [hunk 1]
```diff
@@ -5,7 +5,13 @@ This document contains a list of maintainers in this repo. See [opensearch-proje
...
+|-------------------------|---------------------------------------------| ----------- |
 | Vijayan Balasubramanian | [VijayanB](https://github.com/VijayanB)     | Amazon      |
 | Vamshi Vijay Nakkirtha  | [vamshin](https://github.com/vamshin)       | Amazon      |
+| Junqiu Lei              | [junqiu-lei](https://github.com/junqiu-lei) | Amazon      |
+
+## Emeritus
+
+| Maintainer              | GitHub ID                                   | Affiliation |
+|-------------------------|---------------------------------------------| ----------- |
+| Shivam Dhar             | [Shivamdhar](https://github.com/Shivamdhar) | Amazon      |
```
Comment (martin-gaievski) [inline MAINTAINERS.md:17]: just double-checking, we want to keep Shivam as maintainer, right?

Comment (VijayanB) [reply MAINTAINERS.md:17]: No, as Emeritus.

Comment (junqiu-lei) [reply MAINTAINERS.md:17]: No, but we can move Shivam here. Ref: https://github.com/opensearch-project/.github/blob/main/RESPONSIBILITIES.md#moving-on

Comment (martin-gaievski) [reply MAINTAINERS.md:17]: Got it, didn't know about emeritus section. And how about Affiliation, should we keep Amazon?

File: DEVELOPER_GUIDE.md [hunk 2]
```diff
@@ -21,50 +22,61 @@ You will need to install [node.js](https://nodejs.org/en/), [nvm](https://github
...
-```
-6. Run `yarn osd bootstrap` inside `OpenSearch-Dashboards/plugins/src/plugins/custom_import_map`.
+1. Download OpenSearch [Geospatial](https://github.com/opensearch-project/geospatial) plugin, which requires same version with OpenSearch-Dashboard and maps-dashboards plugin.
+2. Run `./gradlew run` under Geospatial plugin root path to start OpenSearch cluster.
+3. Download the OpenSearch Dashboards source code for the version specified in package.json you want to set up.
+4. Change your node version by `nvm use <version>`to the version specified in `.node-version` inside the OpenSearch Dashboards root directory.
+5. Create a `plugins` directory inside the OpenSearch Dashboards source code directory, if `plugins` directory doesn't exist.
+6. cd into `plugins` directory in the OpenSearch Dashboards source code directory.
+7. Check out this package from version control into the `plugins` directory.
+8. Run `yarn osd bootstrap` inside `OpenSearch-Dashboards/plugins/maps_dashboards` folder.
```
[the remark this answers is not in the document]
Comment (junqiu-lei) [reply DEVELOPER_GUIDE.md]: updated

**Review by martin-gaievski — APPROVED** [review]

**Review by VijayanB — APPROVED** [review]

**Pull request merged**